app/controllers/buddies_controller.rb

```diff
@@ -2,7 +2,7 @@ class  BuddiesController < ApplicationController
   before_action :authenticate_user!, except: [:index, :show]
 
   def index
-    @buddies = Buddy.all
+    @buddies = Buddy.all.page(params[:page]).per(1)
```

Comment:
Yes, we were just testing it with 1 and I thought I changed it to 10 before
pushing. Oops.


On Monday, January 5, 2015, Vikram Ramakrishnan <notifications@github.com>
wrote:

> In app/controllers/buddies_controller.rb
> <https://github.com/cmkoller/code_buddy/pull/27#discussion-diff-22496370>:
>
> > @@ -2,7 +2,7 @@ class  BuddiesController < ApplicationController
> >    before_action :authenticate_user!, except: [:index, :show]
> >
> >    def index
> > -    @buddies = Buddy.all
> > +    @buddies = Buddy.all.page(params[:page]).per(1)
>
> Right now you're paginating 1 Buddy title per page. Is that what you want?
> Something like 10 titles might be more usable.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cmkoller/code_buddy/pull/27/files#r22496370>.
>


-- 
Tracey Hollingsworth
tracey.hollingsworth@gmail.com